cqf-fhir-cr/src/test/resources/org/opencds/cqf/fhir/cr/measure/r4/MinimalMeasureEvaluation/tests/encounter/female-1988-2-encounter-1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "resourceType": "Encounter",
+  "id": "female-1988-encounter-3",
+  "status": "finished",
+  "class": {
+    "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode",
+    "code": "IMP",
+    "display": "inpatient encounter"
+  },
+  "type": [ {
+    "coding": [ {
+      "system": "http://snomed.info/sct",
+      "code": "32485007",
+      "display": "Hospital admission (procedure)"
+    } ]
+  } ],
+  "subject": {
+    "reference": "Patient/female-1988-2"
+  },
+  "period": {
+    "start": "2024-01-29T08:30:00-07:00",
```

Comment:
Suggestion:  Please add "invalid period" or something to that effect in the title of this file.   It's a shame you can't add comments to JSON because it's not obvious at first glance that this JSON contains an invalid period.